src/widget/wspinny.cpp

```diff
@@ -318,6 +318,8 @@ void WSpinny::render(VSyncThread* vSyncThread) {
         return;
     }
 
+    makeCurrent();
```

Comment:
IIRC this is expensive. Everywhere else we use this:

```cpp
if (QGLContext::currentContext() != context()) {
    makeCurrent();
}
```

Reply:
I tried that but it makes no difference. The condition is always false.